### PR TITLE
feat: npmPackageVersion option

### PR DESCRIPTION
**Pull request by PierreDemailly**
This PR add a new `version` option that allow to fetch the Scorecard result for a specific version (when given a npm package only)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -91,6 +91,11 @@ export interface IResultOptions {
    * @default true
    */
   resolveOnVersionControl?: boolean;
+  /**
+   * @description The version of the npm package (when `resolveOnNpmRegistry` only) to retrieve the scorecard for.
+   * @default "latest"
+   */
+  version?: string;
```
Comment (fraxken) [inline README.md:98]: version feel like a bit too simplistic no? Like version of what?

`npmPackumentVersion` or something like that instead?

Comment (PierreDemailly) [reply README.md:98]: Thought the doc + jsdoc would be enough to make it clear but I agree `version` is not explicit.
`packageVersion` ?
or `npmPackageVersion` if we wanna be explicit that this option is for npm only.
`npmPackumentVersion` could represents the version of packument itself, no ?

Comment (fraxken) [reply README.md:98]: npmPackageVersion is fine

Comment (PierreDemailly) [reply README.md]: Done

**Review by fraxken — APPROVED** [review]

**Pull request merged**